acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>Suggest to attach the download script for convenience.</p><p><code></code><code>python</p><p>import os</p><p>import tempfile</p><p>from monai.apps import download_and_extract</p><p> </p><p>resource = "https://msd-for-monai.s3-us-west-2.amazonaws.com/Task09_Spleen.tar"</p><p>md5 = "410d4a301da4e5b2f6f86ec3ddba524e"</p><p> </p><p>directory = os.environ.get("MONAI_DATA_DIRECTORY")</p><p>root_dir = tempfile.mkdtemp() if directory is None else directory</p><p>compressed_file = os.path.join(root_dir, "Task09_Spleen.tar")</p><p>data_dir = os.path.join(root_dir, "Task09_Spleen")</p><p>if not os.path.exists(data_dir):</p><p>    download_and_extract(resource, compressed_file, root_dir, md5)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>